plugins/certs/help-functions

```diff
@@ -29,6 +29,7 @@ fn-help-content() {
   cat <<help_content
     certs:add <app> CRT KEY, Add an ssl endpoint to an app. Can also import from a tarball on stdin
     certs:chain CRT [CRT ...], [NOT IMPLEMENTED] Print the ordered and complete chain for the given certificate
+    certs:download <app> key|crt, Print the server.key or server.crt on stdout
```

Comment:
Should be `[key|crt]`.

Reply:
My understanding is that a `[]` block meant optional in documentation. I've changed it though.